func/mpi/mpi_wincreate.cpp

```diff
@@ -80,12 +80,17 @@ FAASM_MAIN_FUNC()
         }
 
         // Check size of window
-        checkIntAttr(window, MPI_WIN_SIZE, winSize, "MPI_WIN_SIZE");
-        checkIntAttr(window, MPI_WIN_DISP_UNIT, 1, "MPI_WIN_DISP_UNIT");
+        if (!checkIntAttr(window, MPI_WIN_SIZE, winSize, "MPI_WIN_SIZE"))
+            return 1;
+        if (!checkIntAttr(window, MPI_WIN_DISP_UNIT, 1, "MPI_WIN_DISP_UNIT"))
+            return 1;
```

Comment:
We never do if statements without curly brackets, it's a consistent source of nasty bugs. 

I'm surprised this isn't captured in the code style